### ISSUE TITLE
resolve conflicts "automatically"

### ISSUE THREAD

**Issue by Andrew-Colman** [Andrew-Colman/sync-fork#12]
--reset (reset the fork to be identical to current remote )

**Comment by FarazzShaikh** [comment]
I'll give this a shot. Also what do you mean by --force? Also do you mean to reset all changes with --reset?

**Comment by Andrew-Colman** [comment]
> I'll give this a shot. Also what do you mean by --force?

Apply both changes (local files and remote files) (but not recommended) 

```js
============current
const js = true;
=================
const js = false;
=============master
```

```js
const js = true;
const js = false;
```


maybe something from https://git-scm.com/docs/git-rebase ? 


<hr>

> Also do you mean to reset all changes with --reset?

like the project was just forked and cloned, basically reset 

maybe this command https://git-scm.com/docs/git-reset 
`git reset --hard <upstream>`

**Comment by FarazzShaikh** [comment]
> Apply both changes (local files and remote files) (but not recommended)

Won't this cause a ton of errors?

I'll get started with -upcoming and -local, it'll just be the -theirs and -ours options in `git merge -X [theirs/ours] <branch>`. And of course -reset

**Comment by Andrew-Colman** [comment]
>Won't this cause a ton of errors?

yeah, It's not a good idea, 

>I'll get started with -upcoming and -local, it'll just be the -theirs and -ours options in git merge -X [theirs/ours] <branch>. And of course -reset

for the simplicity and purpose of the package this will work

**Comment by FarazzShaikh** [comment]
Closed by #16
